src/pages/Vai/styles.ts

```diff
@@ -14,7 +14,7 @@ export const useStyles = () => {
       flex-direction: column;
     `,
     tabs: css`
-      max-width: ${theme.spacing(136)};
+      max-width: ${theme.spacing(140)};
```

Comment:
@therealemjy this change seems random, but it comes from adding the new tooltip, I found that it looks better with a bit more width, let me know if that is fine.

Reply:
Ok sounds good, thanks for adding context